src/core/varinfo.jl

```diff
@@ -169,56 +171,50 @@ end
 
 # Main behaviour control of rand() depending on sampler type and if sampler inside
 rand(vi::VarInfo, vn::VarName, dist::Distribution, spl::Sampler, inside=true) = begin
```

Comment:
since `inside` needs to be tested for each `rand` call, maybe it makes sense to remove the `inside` parameter, and move the code testing the association of a variable and a sampler into `rand`?